package.json

```diff
@@ -7,9 +7,9 @@
   },
   "scripts": {
     "build": "tsc && NODE_ENV=production nodemon src/server/server.ts",
-    "start": "tsc && NODE_ENV=development nodemon src/server/server.ts",
```

Comment:
No need to revert if it will make Windows user's work easier. I think it's fine for now since `NODE_ENV=development` doesn't really affect the code that much.